exercise4/isPrime.js

```diff
@@ -0,0 +1,9 @@
+function isPrime(number) {
+  if (number < 2) return false;
+  if (number == 2) return true;
+
```

Comment:
[OPTIONAL] You can also remove this line of code since in the next line where you are looping you start from `2`, It will handle it for you.

Reply:
Yeah sure